client/src/utils/constants.js

```diff
@@ -170,6 +170,13 @@ export const botConfigs = [
         description: 'Older or hacked Switches sometimes take longer loading the title screen. You\'ll know this is happening if the title screen plays for a long time before entering the game. If this happens to you, check this box, and the bot will wait a little longer before pressing A to skip the title screen.',
         type: 'bool',
         value: 'false',
+      },
+	  {
+        param: 'm_pokemonIsCatchable',
```

Comment:
This param type needs to be added in `/utils/constants.js` so that the server knows what type to make this variable when it overwrites the config